reorganize readme, to match timeline of 469

elkolorado opened the pull request:
Hey, I reorganized the structure of file, by it only FACT inside game, and by time that they occured. The list is long as you can see from 2001-2021. It does show that cipsoft remember bout 469 and contantly add more information about it. You can then add your theories previously added into readme.md to Issues, as we all do.

s2ward commented:
I think this is a good idea, however I think it would fit better in a separate .md for chronological release.
Current README.md is ordered in a deliberate way so that people without knowledge about the game can have a go at it.
I would also not want to add the sum of research papers, because it is way too farfetched - especially as it does not take into account all of the papers.

Some reorganizing to the README could be done, I like how the wrinkled bonelord dialogue comes early on.
I would also like to add your find about '07' as mentioned in [issue 13](https://github.com/s2ward/469/issues/13).

How would feel about adding a link to the chronological readme in the current README?

elkolorado commented:
> I think this is a good idea, however I think it would fit better in a separate .md for chronological release. Current README.md is ordered in a deliberate way so that people without knowledge about the game can have a go at it. I would also not want to add the sum of research papers, because it is way too farfetched - especially as it does not take into account all of the papers.
> 
> Some reorganizing to the README could be done, I like how the wrinkled bonelord dialogue comes early on. I would also like to add your find about '07' as mentioned in [issue 13](https://github.com/s2ward/469/issues/13).
> 
> How would feel about adding a link to the chronological readme in the current README?

sure

elkolorado commented:
@s2ward check out the latest commits, I've moved the timeline to separate file, and also added a little json & score matrix analysis for your alignment idea
